cmd/migrations/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"database/sql"
+	"flag"
+	"fmt"
+
+	"github.com/go-devs-ua/octagon/app/repository/pg"
+	"github.com/go-devs-ua/octagon/lgr"
+	_ "github.com/lib/pq"
+
+	migrate "github.com/rubenv/sql-migrate"
+)
+
+func main() {
+	logger := lgr.New()
+
+	opt, err := pg.GetConfig()
+	if err != nil {
+		logger.Errorf("Failed to get config from .env: %+v\n", err)
+		return
+	}
+
+	db, err := pg.ConnectDB(opt.DB)
+	if err != nil {
+		logger.Errorf("%+v\n", err)
+		return
+	}
+
+	direction := flag.String("migrate", "", "applying migrations 'up/down'")
+	flag.Parse()
+
+	var dir migrate.MigrationDirection
```

Comment:
We keep migrations related logic in main(). Let's try to avoid this.

You need to pass to `migrateDB()` `up` or `down` and inside it you can do this conversion to 0 or 1.
So, this should be encapsulated inside `migrateDB()`

Reply:
Yeah, totally makes sense, fixed.